Source/Core/DolphinQt2/HotkeyScheduler.cpp

```diff
@@ -261,50 +262,55 @@ void HotkeyScheduler::Run()
       if (IsHotkey(HK_INCREASE_IR))
       {
         show_msg(OSDMessage::IRChanged);
-        ++g_Config.iEFBScale;
+        Config::SetCurrent(Config::GFX_EFB_SCALE, Config::Get(Config::GFX_EFB_SCALE) + 1);
       }
       if (IsHotkey(HK_DECREASE_IR))
       {
         show_msg(OSDMessage::IRChanged);
-        g_Config.iEFBScale = std::max(g_Config.iEFBScale - 1, EFB_SCALE_AUTO_INTEGRAL);
+        if (Config::Get(Config::GFX_EFB_SCALE) > EFB_SCALE_AUTO_INTEGRAL)
```

Comment:
`Config::Get(Config::GFX_EFB_SCALE)` should probably be stored to a variable, so a config lookup doesn't need to be done twice (here and in the code within the if statement).